test/wasm-only.fromasm

```diff
@@ -266,6 +266,32 @@
    )
```

Comment:
what are the three binary files added? (`test/validator/*.wasm`)
(this isn't the right place for this comment, but it's the closest)

Reply:
I think those are temporary `./check.py` files that I accidentally added with `git add test/` in my excitement that this worked. Will remove.